pyaedt/hfss.py

```diff
@@ -2982,3 +2983,319 @@ def assign_radiation_boundary_to_faces(self, faces_id, boundary_name=""):
+                doppler_ad_sampling_rate, "MHz")
+        setup1.update()
+        return setup1
+
+    @aedt_exception_handler
+    def _create_sbr_doppler_sweep(self, setupname, time_var, tstart, tstop, tsweep,parametric_name):
+        time_start = self.modeler.primitives._arg_with_dim(tstart, "sec")
+        time_sweep = self.modeler.primitives._arg_with_dim(tsweep, "sec")
+        time_stop = self.modeler.primitives._arg_with_dim(tstop, "sec")
+        sweep_range = "LIN {} {} {}".format(time_start, time_stop, time_sweep)
+        return self.opti_parametric.add_parametric_setup(time_var, sweep_range, setupname,
+                                                         parametricname=parametric_name)
+
+    @aedt_exception_handler
+    def create_sbr_chirp_i_doppler_setup(self, time_var=None, sweep_time_duration=0, center_freq=76.5, resolution=1, period=200,
+                                         velocity_resolution=0.4, min_velocity=-20, max_velocity=20,
+                                         ray_density_per_wavelenght=0.2, max_bounces=5, include_coupling_effects=False,
+                                         doppler_ad_sampling_rate=20, setup_name=None):
+        """Create an SBR+ Chirp IQ Setup.
+
+        Parameters
+        ----------
+        time_var: str, Optional
+            Name of the time variable. Default ``None`` which will search for first Time Variable available.
+        sweep_time_duration: float, Optional
+            Sweep Time Duration. If greater than 0, a parametric sweep will be created. Default ``0``.
+        center_freq: float, Optional
+            Center Frequency in GHz. Default ``76.5``.
+        resolution: float, Optional
+            Doppler Resolution in meter. Default ``1``.
+        period: float, Optional
+            Period of Analysis in meter. Default ``200``.
+        velocity_resolution: float, Optional
+            Doppler Velocity Resolution in m_per_sec. Default ``0.4``.
+        min_velocity: str, Optional
+            Minimum Doppler Velocity in m_per_sec. Default ``-20``.
+        max_velocity: str, Optional
+            Maximum Doppler Velocity in m_per_sec. Default ``20``.
+        ray_density_per_wavelenght: float, Optional
+            Doppler Ray Density per wavelength. Default ``0.2``.
+        max_bounces: int, Optional
+            Maximum number of Bounces. Default ``5``.
+        include_coupling_effects: float, Optional
+            Set if Coupling Effects will be included. Default ``False``.
+        doppler_ad_sampling_rate: float, Optional
+            Doppler AD Sampling Rate. It works only if include_coupling_effects is ``True``. Default ``20``.
+        setup_name: str, Optional
+            Name of the Setup. Default ``None``.
+
+        Returns
+        -------
+        (:class:`pyaedt.modules.SolveSetup.Setup`, :class:`pyaedt.modules.DesignXPloration.Optimetrics.ParametericsSetups`)
```

Comment:
Since we're using the `numpydoc` extension, we no longer need to worry about including the `:class:` directive

```suggestion
        pyaedt.modules.SolveSetup.Setup
            Solve setup.
        pyaedt.modules.DesignXPloration.Optimetrics.ParametericsSetups
            Parametric setup.
```